lib/liftoff/scheme_builder.rb

```diff
@@ -0,0 +1,53 @@
+module Liftoff
+  class SchemeBuilder
+    def initialize(xcode_project, config)
+      @xcode_project = xcode_project
+      @config = config
+    end
+
+    def create_schemes
+      @xcode_project.generate_default_scheme
```

Comment:
What do you think about making `@xcode_project` and `@config` private `attr_readers` and using that throughout this class to access them?

We have an open PR for our guides that recommends this that looks like it's probably going to be merged. https://github.com/thoughtbot/guides/pull/331